packages/design-system/src/components/Table/Table.jsx

```diff
@@ -69,18 +69,25 @@ export class Table extends React.PureComponent {
     const { scrollWidth, clientWidth } = this.container;
     const isScrollActive = scrollWidth > clientWidth;
```

Comment:
Small styling preference, but one way this could be simplified is:
```
const scrollActive = scrollWidth > clientWidth;
this.setState({ scrollActive });
```